src/Mod/Start/Gui/FileCardDelegate.cpp

```diff
@@ -22,6 +22,7 @@
  ***************************************************************************/
 
 #include "PreCompiled.h"
+#include <qstringliteral.h>

```

Comment:
Should it be there? Looks like vscode auto included that file as it does not follow the typical style of Qt imports. It should also be in `_PreComp_` section if needed. I'd suggest simply including `<QString>` here as this is the header mentioned by the docs